mapp/0.0.9/Dockerfile

```diff
@@ -0,0 +1,52 @@
+###############################################################################
+#
+#   Dockerfile for the container
+#
+#   AUTHOR: Maciej_Bak
+#   AFFILIATION: Swiss_Institute_of_Bioinformatics
```

Comment:
Unless you are paid by SIB, I think you should mention Biozentrum / Uni Basel here as well. Whoever pays you is the copyright holder!